test/run_envoy_coverage.sh

```diff
@@ -6,9 +6,10 @@ PROJECT_SOURCE_DIR=$3
 PROJECT_BINARY_DIR=$4
 GCOVR=$5
 GCOVR_EXTRA_ARGS=$6
+EXTRA_SETUP_SCRIPT=$7
```

Comment:
Coverage is broken on our private build without this change, I just fixed it here. It's related to other test PRs that @htuch has been doing.